modules/ingestor/src/graph/sbom/cyclonedx.rs

```diff
@@ -457,10 +457,7 @@ impl<'a> ComponentCreator<'a> {
                             continue 'l;
                         };
 
-                        let license = LicenseInfo {
-                            license,
-                            refs: Default::default(),
-                        };
+                        let license = LicenseInfo { license };
```

Comment:
To my understanding, in the case of the `id` being used, we'd need to resolve that first into a name? Right?